cli/crates/cli/src/output/report.rs

```diff
@@ -69,6 +72,20 @@ pub fn error(error: &CliError) {
     }
 }
 
+pub fn warnings(warnings: &[Warning]) {
+    for warning in warnings {
+        let msg = warning.message();
+
+        watercolor::output!("Warning: {msg}", @BrightYellow);
+
+        if let Some(hint) = warning.hint() {
+            watercolor::output!("Hint: {hint}", @BrightBlue)
+        }
+
+        println!();
```

Comment:
We could just add a `\n` after the hint

Reply:
but if hint is not there, we need to add it to the warning